cli/tests/076_websocket_no_permission.js.out

```diff
@@ -0,0 +1 @@
+warning: network access to "ws://localhost/", run again with the --allow-net flag
```

Comment:
This is a bit surprising, is there a reason why we can't catch this error? What happens if there's `onerror` handler attached to the WebSocket? It'd be great to test that scenario as well

Reply:
The error will be passed to the error event as well